CONTRIBUTING.md

```diff
@@ -0,0 +1,17 @@
+# Contributing to bevy_libgdx_atlas
+
+We happily welcome folks to contribute in some way. This document will help you in contributing, however you'll do that.
+
+## Reporting issues and requesting features
+
+Please report issues and request features at the relevant sections of this repository. For issues please state what you expected to happen, what went wrong, and any other extra info that you feel is relevant. For PRs please state the objective, how you are solving that objective with this PR, and how you have tested it / how reviewers may test it to confirm it works.
+
+Check for duplicates before submitting, please.
+
+## Testing
+
+To test `bevy_libgdx_atlas` in the same way that the continuous integration does, run:
+
+`cargo clippy -- -Dwarnings && cargo test --verbose && cargo doc --all-features --no-deps --document-private-items --keep-going && cargo fmt --check` 
```

Comment:
lets provide a justfile for that as a simple alternative (fine as a followup, but should be on our checklist for all our crates to have)